.github/workflows/markdown-checks.yaml

```diff
@@ -29,3 +29,12 @@ jobs:
           else
             echo "JSON files up to date. Looks good!"
           fi
+
+  markdown-lint:
```

Comment:
```suggestion
  json-lint:
```

Reply:
I didn't realize this was linting the JSON. Is there any point to lint generated files at all?

Reply:
It might be a technically easier way to validate the normative statements, but to your point, we could embed that functionality directly into the parser.

Reply:
to validate them how. are the statements themselves checked by the linter and not just the format/spacing?

Reply:
> to validate them how. are the statements themselves checked by the linter and not just the format/spacing?

Yes - it checks that an RFC2119 keyword is present, for instance.

Reply:
no lint is probably a fine name for that